v2/datastream-to-spanner/src/test/java/com/google/cloud/teleport/v2/templates/DataStreamToSpannerLT100Gb.java

```diff
@@ -0,0 +1,197 @@
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.templates;
+
+import static org.apache.beam.it.truthmatchers.PipelineAsserts.assertThatPipeline;
+import static org.apache.beam.it.truthmatchers.PipelineAsserts.assertThatResult;
+
+import com.google.cloud.datastream.v1.Stream;
+import com.google.cloud.teleport.metadata.TemplateLoadTest;
+import com.google.pubsub.v1.SubscriptionName;
+import java.io.IOException;
+import java.text.ParseException;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.beam.it.common.PipelineLauncher;
+import org.apache.beam.it.common.PipelineLauncher.LaunchConfig;
+import org.apache.beam.it.common.PipelineOperator;
+import org.apache.beam.it.common.TestProperties;
+import org.apache.beam.it.common.utils.ResourceManagerUtils;
+import org.apache.beam.it.conditions.ConditionCheck;
+import org.apache.beam.it.gcp.datastream.DatastreamResourceManager;
+import org.apache.beam.it.gcp.datastream.MySQLSource;
+import org.apache.beam.it.gcp.pubsub.PubsubResourceManager;
+import org.apache.beam.it.gcp.secretmanager.SecretManagerResourceManager;
+import org.apache.beam.it.gcp.spanner.SpannerResourceManager;
+import org.apache.beam.it.gcp.spanner.conditions.SpannerRowsCheck;
+import org.apache.beam.it.gcp.storage.GcsResourceManager;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Performance tests for {@link DataStreamToSpanner} DataStream to Spanner template. */
+@Category(TemplateLoadTest.class)
+@TemplateLoadTest(DataStreamToSpanner.class)
+@RunWith(JUnit4.class)
+public class DataStreamToSpannerLT100Gb extends DataStreamToSpannerLTBase {
+
+  private static final String SPEC_PATH =
+      "gs://dataflow-templates/latest/flex/Cloud_Datastream_to_Spanner";
+  private static final String ARTIFACT_BUCKET = TestProperties.artifactBucket();
+  private static final String TEST_ROOT_DIR = DataStreamToSpannerLT100Gb.class.getSimpleName();
+  private static final String SPANNER_DDL_RESOURCE =
+      "DataStreamToSpannerLT100Gb/spanner-schema.sql";
+  private static final String TABLE = "person";
+  private static final int MAX_WORKERS = 100;
+  private static final int NUM_WORKERS = 50;
+  public static PubsubResourceManager pubsubResourceManager;
+  public static SpannerResourceManager spannerResourceManager;
+  private static GcsResourceManager gcsResourceManager;
+  public static DatastreamResourceManager datastreamResourceManager;
```

Comment:
What is the benefit of declaring them `static`?

Reply:
These need not be static. Corrected it. I took some of this code from IT classes and forgot to remove static. Corrected it in the Reverse replication PR as well.